src/EA.Weee.Web/Views/Shared/_ViewEvidenceNotePartial.cshtml

```diff
@@ -280,3 +282,12 @@ else
 
     </tbody>
 </table>
+
+@if (Model.IsPrintable)
+{
+    @(this.WeeeGds().CreateProgressiveDisclosure("Transfer history",
```

Comment:
@GuidoWorkDev We dont want to output this as a progressive disclosure in the PDF as it just needs to be a table. Have you tried this locally compared to the mockup?

You should just be able to all @Html.Partial("~/Views/Shared/_EvidenceTransferProgressiveDisclosurePartial.cshtml") passing the model...